conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptions.java

```diff
@@ -191,12 +191,21 @@ private static boolean qosExceptionHasAdditionalMetadata(QosException qosExcepti
     // error types don't propagate through other services.
     private static void remoteException(HttpServerExchange exchange, RemoteException remoteException) {
         if (remoteException.getStatus() == 401 || remoteException.getStatus() == 403) {
-            log.info(
-                    "Encountered a remote exception",
-                    SafeArg.of("errorInstanceId", remoteException.getError().errorInstanceId()),
-                    SafeArg.of("errorName", remoteException.getError().errorName()),
-                    SafeArg.of("statusCode", remoteException.getStatus()),
-                    remoteException);
+            if (remoteException.getStatus() == 401 && log.isDebugEnabled()) {
```

Comment:
Wouldn't this still log 401s at INFO? Do we want:

```suggestion
            if (remoteException.getStatus() == 401 || log.isDebugEnabled()) {
```